cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java

```diff
@@ -90,20 +92,35 @@ public boolean isSimpleType() {
    * Logical type TIMESTAMP_MICROS relies on LONG as underlying primitive type
    * Logical type TIME_MILLIS relies on INT as underlying primitive type
    * Logical type TIME_MICROS relies on LONG as underlying primitive type
+   * Logical type DECIMAL relies on BYTES as underlying primitive type
    *
    * For example, the json schema for logicaltype date will be as below:
    * {
    *   "type" : "int",
    *   "logicalType" : "date"
    * }
    *
+   * Example schema for logicaltype decimal will be as below:
+   * {
+   *   "type" : "bytes",
+   *   "logicalType" : "decimal",
+   *   "precision" : precision,
+   *   "scale" : scale
+   * }
+   * where precision is an int representing maximum number of unscaled digits in a decimal and
+   * positive scale is an int representing number of digits to the right of decimal number whereas
+   * negative scale is an int representing number of zeros appended to decimal number.
+   *
+   * For example,
+   * 10.2 number will have precision as 3 as scale as 1.
```

Comment:
nit: I think it would help to have one more example so that the reader can more easily see how the values change and why.

Reply:
added an example for negative scale check.